grammars/rescript.tmLanguage.json

```diff
@@ -516,10 +512,6 @@
     {
       "include": "#jsx"
     },
-    {
-      "include": "#jsx-attributes"
```

Comment:
My bad, I tried to also include some jsx highlight - but i just gave up, and forget to remove it 😅